rotkehlchen/history/price.py

```diff
@@ -173,6 +175,9 @@ def get_price_for_special_asset(
                 timestamp=timestamp,
             )
 
+        if from_asset.is_evm_token() and (pool_token := from_asset.resolve_to_evm_token()).protocol in {UNISWAP_PROTOCOL, UNISWAPV3_PROTOCOL}:  # noqa: E501
+            return PriceHistorian.query_uniswap_position_price(pool_token, ONE, to_asset, timestamp)  # noqa: E501
```

Comment:
another option that you have is instead of raising errors catch them and raise None since in the docstring of this function you have

```
Can return None if the from asset is not in the list of special cases
```

however you handle it please document it properly in the docstring